moduleroot/Rakefile.erb

```diff
@@ -36,7 +36,7 @@ end
 
 def changelog_project
   return unless Rake.application.top_level_tasks.include? "changelog"
-  returnVal = <%= @configs['changelog_project'].inspect -%> || JSON.load(File.read('metadata.json'))['source'].match(%r{.*/([^/]*)})[1]
+  returnVal = <%= @configs['changelog_project'].inspect -%> || JSON.load(File.read('metadata.json'))['source'].match(%r{.*/([^/]*)})[1].reverse.sub(%r{tig\.}, '').reverse
```

Comment:
```suggestion
  returnVal = <%= @configs['changelog_project'].inspect -%> || JSON.load(File.read('metadata.json'))['source'].match(%r{.*\/([^\/]*?)(?:\.git)?\Z})[1]
```

I think something like this would allow you to get rid of the substitutions at the end.